Dockerfile

```diff
@@ -27,46 +9,46 @@ RUN apt-get update && apt-get install -y \
 
 RUN pip3 install --upgrade pip setuptools wheel
 
+ARG DEBIAN_FRONTEND=noninteractive
```

Comment:
Do we want to keep this arg at the top? I think it comes into play with the `apt-get install` step

Reply:
Oh good catch, this doesn't seem to impact the build at all but I've re-ordered the steps to match what was being run originally.